### PR TITLE
#team info db

### PR DESCRIPTION
**Pull request by DysonBreakstone**
This adds the #team_info method from iteration 4 and alters the Team class to have the attributes needed for it.

### CODE REVIEW

File: lib/team.rb [hunk 1]
```diff
@@ -3,13 +3,15 @@ class Team
               :franchiseid, 
               :teamname, 
               :abbreviation, 
-              :stadium
+              :stadium,
```
Comment (margdaly) [inline lib/team.rb:6]: do we need stadiums in any other method? should it just be removed as an attribute altogether from team_data?

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -117,46 +117,59 @@
...
   describe "Tackles" do
-    it "#most_tackles" do
+    xit "#most_tackles" do
       expect(@stat_tracker.most_tackles("20132014")).to eq "FC Cincinnati"
       expect(@stat_tracker.most_tackles("20142015")).to eq "Seattle Sounders FC"
     end
 
-    it "#fewest_tackles" do
+    xit "#fewest_tackles" do
+    require 'pry'; binding.pry
```
Comment (margdaly) [inline spec/stat_tracker_spec.rb:156]: have to remove this!

File: spec/stat_tracker_spec.rb [hunk 3]
```diff
@@ -117,46 +117,59 @@
...
 
   describe "Tackles" do
-    it "#most_tackles" do
+    xit "#most_tackles" do
       expect(@stat_tracker.most_tackles("20132014")).to eq "FC Cincinnati"
       expect(@stat_tracker.most_tackles("20142015")).to eq "Seattle Sounders FC"
     end
 
-    it "#fewest_tackles" do
+    xit "#fewest_tackles" do
```
Comment (margdaly) [inline spec/stat_tracker_spec.rb:155]: are these all meant to be skipped?

**Review by margdaly — COMMENTED** [review]
Do we need @stadium for any other method or can we swap that out for links?

File: spec/stat_tracker_spec.rb [hunk 4]
```diff
@@ -17,57 +17,57 @@
   end
   
   describe 'initialize' do
-    it 'exists' do
+    xit 'exists' do
```
Comment (Jredish11) [inline spec/stat_tracker_spec.rb:20]: reminder to un 'x' these test before final pr

**Review by Jredish11 — APPROVED** [review]

File: spec/stat_tracker_spec.rb (hunk 4 above)
Comment (DysonBreakstone) [reply spec/stat_tracker_spec.rb:20]: Good call!

File: spec/stat_tracker_spec.rb (hunk 3 above)
Comment (DysonBreakstone) [reply spec/stat_tracker_spec.rb:155]: My bad! I'll unskip them on the next push

File: spec/stat_tracker_spec.rb (hunk 2 above)
Comment (DysonBreakstone) [reply spec/stat_tracker_spec.rb:156]: Righto!

File: lib/team.rb (hunk 1 above)
Comment (DysonBreakstone) [reply lib/team.rb:6]: I honestly don't think we do, I left them in because I didn't want to break any tests

**Review by AlejandroLopez1992 — APPROVED** [review]

File: spec/stat_tracker_spec.rb (hunk 3 above)
Comment (margdaly) [reply spec/stat_tracker_spec.rb:155]: no worries at all! 
Great work on starting iteration 4!

File: lib/team.rb (hunk 1 above)
Comment (margdaly) [reply lib/team.rb:6]: I can go and check that everything still runs with it's removal and push those changes form a new branch!

File: spec/stat_tracker_spec.rb (hunk 2 above)
Comment (margdaly) [reply spec/stat_tracker_spec.rb:156]: Thank you!!

**Pull request merged**